Sources/TuistCore/Xcode/XcodeController.swift

```diff
@@ -34,4 +41,25 @@ public class XcodeController: XcodeControlling {
         }
         return try Xcode.read(path: AbsolutePath(path).parentDirectory.parentDirectory)
     }
+
+    enum XcodeVersionError: Swift.Error {
```

Comment:
It should extend `FatalError`

Reply:
True 🙂 changed.

Reply:
I still see this error conforming `Swift.Error`